### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @BetterBlood

### DISCUSSION THREAD

**Discussion by BetterBlood** [heig-vd-dai-course/heig-vd-dai-course#49] [Show and tell]
[The link to _your_ GitHub repository](https://github.com/BetterBlood/maven-java)

**Comment by ludelafo** [comment]
Thank you!

I would add the command to download the dependencies as well so a new developer can build and run your app "from scratch".

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
